ui-components/src/forms/Selector.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import Select from "react-select"
+
+export interface SelectorProps {
+  options?: { value: string; label: string }[]
+  closeMenu?: boolean
+  defaultValue?: { value: string; label: string }[]
+  multiSelect?: boolean
+}
+
+const Selector = (props: SelectorProps) => {
+  return (
+    <div style={{ width: 500 }}>
```

Comment:
Can you move this style to a tailwind style, or a class in forms.scss: https://github.com/CityOfDetroit/bloom/blob/32d02d2618f626a4a2a89c97f94be9d13c0c5a6c/ui-components/src/global/forms.scss

That will set us up better for future changes to styles as we work on this.